README.md

```diff
@@ -57,32 +59,32 @@ The items in arrays can be **ASCII code** of keys or **characters itself**.
 
 #### Example values of detectKeys array
 
-```
+```json
 {
 	detectKeys: ['A', 69, 27]
```

Comment:
Please add double quotes for json keys and strings in code blocks to avoid errors.
e.g in the next comment